src/components/infotekst/Infotekst.tsx

```diff
@@ -57,6 +57,9 @@ function Infotekst(props: HjelpetekstProps) {
                       {text}
                     </>
                   ),
+                  b: (text: string) => (
+                      <b>{text}</b>
```

Comment:
Bruk av <b> er ikke anbefalt.
Bør bruke designsystem-kompenenter, men vet ikke om det vil funke inne i FormattedMessage?

Reply:
Dette er gjort for å tillate bold-tags i tekst-variablene i nb.ts/en.ts. Bold font på overskriftene er lagt inn etter anbefalinger fra Chris. Har avklart med Ole Jonas at dette kan prodsettes slik i første omgang, så får vi evt ta justeringer senere.